server/controllers/stock/reports/stock_consumption_graph.handlebars

```diff
@@ -0,0 +1,80 @@
+{{> head title="TREE.STOCK_CONSUMPTION_GRAPH_REPORT" }}
+
+<script src="{{nodeModulesPath}}/chart.js/dist/Chart.min.js"></script>
+
+<body>
+
+    <div class="container">
+        {{> header}}
+
+        <!-- body  -->
+        <div class="row">
+            <div class="col-xs-12">
+
+                <!-- page title  -->
+                <h2 class="text-center">
+                    {{translate 'REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.TITLE'}}
+                </h2>
+
+                {{#if depot.text}}
+                     <h4 class="text-center">{{depot.text}}</h4>
+                {{/if}}
+                <h4 class="text-center">
+                    {{date dateFrom}} - {{date dateTo}}
+                </h4>
+            </div>
+            <canvas id="myChart" width="600" height="200"></canvas>
+           
+        </div>
+    </div>
+
+
+
+    <script>
+        window.addEventListener('load', function() {
+            var ctx = document.getElementById('myChart').getContext('2d');
+            var chart = new Chart(ctx, {
+                // The type of chart we want to create
+                type: 'bar',
+
+                // The data for our dataset
+                data: {
+                    labels:  {{{labels}}},
+                    datasets: [{
+                        label: {{{title}}},
+                        backgroundColor: 'rgb(255, 99, 132)',
+                        borderColor: 'rgb(255, 99, 132)',
+                        data: {{{data}}}
+                    }]
+                },
+
+                // Configuration options go here
+                options: {
+                    animation: {
+                        duration : 0,
```

Comment:
This is a good start. Let's disable all animations.  The documentation can be found [here](https://www.chartjs.org/docs/latest/general/performance.html#disable-animations):
```js
    options: {
        animation: {
            duration: 0 // general animation time
        },
        hover: {
            animationDuration: 0 // duration of animations when hovering an item
        },
        responsiveAnimationDuration: 0 // animation duration after a resize
    }
```